plugins/uefi-dbx/meson.build

```diff
@@ -12,7 +12,11 @@ plugin_builtin_uefi_dbx = static_library('fu_plugin_uefi_dbx',
   include_directories: plugin_incdirs,
   link_with: plugin_libs,
   c_args: cargs,
-  dependencies: plugin_deps,
+  dependencies: [
+    plugin_deps,
+    efiboot,
+    efivar,
```

Comment:
just as a nit to avoid the same thing in 3 places in one file, can you instead do something like:

`uefi_dbx_deps = [plugin_deps, efiboot, efivar]`

And then use `dependencies: uefi_dbx_deps`?

Reply:
I don't think we want to use more of efiboot. Do we know the format of Boot####? Is it simple to parse using glib in this common case?

Reply:
Trying to get them with glib.

Reply:
@CragW how did you get on? Is this PR still on your radar?